### ISSUE TITLE
The release v1.21.6 failed

### ISSUE THREAD

**Issue by suzuki-shunsuke** [stripe/stripe-cli#1252]
No asset for Linux was released.

https://github.com/stripe/stripe-cli/releases/tag/v1.21.6

<img width="356" alt="image" src="https://github.com/user-attachments/assets/cce610c0-2f6d-47d4-a319-e2a86036917a">

The release workflow failed.

https://github.com/stripe/stripe-cli/actions/runs/11018824575/job/30602654832

GoReleaser failed.
Seems like pushing a docker image to Docker Hub failed.

```
   ⨯ release failed after 72.74s error=docker images: failed to publish artifacts: failed to push stripe/stripe-cli:latest-amd64: exit status 1: The push refers to repository [docker.io/stripe/stripe-cli]
11e7a0b72a51: Preparing
8658df255ad3: Preparing
63ca1fbb43ae: Preparing
denied: requested access to the resource is denied

Error: The process '/opt/hostedtoolcache/goreleaser-action/0.184.0/x64/goreleaser' failed with exit code 1
```

**Comment by suzuki-shunsuke** [comment]
Docker Login Succeeded.

https://github.com/stripe/stripe-cli/actions/runs/11018824575/job/30602654832

```
Run echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin
WARNING! Your password will be stored unencrypted in /home/runner/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded
```

**Comment by charliecruzan-stripe** [comment]
Hi! we're aware of this and looking into it

**Comment by charliecruzan-stripe** [comment]
hi! this is fixed now, and we reran the release for 1.21.7 so you should all be unblocked for now :) I'm leaving 1.21.6 as-is but if, for some reason, someone needs that version instead of the newer version (1.21.7), let me know and i can help you out
